internal/characters/yaoyao/asc.go

```diff
@@ -62,12 +62,13 @@ func (c *char) a1Throw() {
 
 func (c *char) a4(index, src int) func() {
 	return func() {
-		if c.a4Srcs[index] != c.Core.F {
-			return
-		}
+		// TODO: Check if reapplying radish will reset heal timer or not?
+		// if c.a4Srcs[index] != src {
+		// 	return
+		// }
```

Comment:
what's wrong?

Reply:
A4 doesn't heal currently. For this? I don't know if applying another A4 will reset the timing for the next A4 healing tick. I assume not but it's not tested.

Reply:
https://github.com/genshinsim/gcsim/pull/1244#discussion_r1096565434
uncomment it